shims/spark311/src/main/scala/com/nvidia/spark/rapids/shims/spark311/SparkShimServiceProvider.scala

```diff
@@ -22,7 +22,9 @@ object SparkShimServiceProvider {
   // keep 3.1.0 snapshot version for now until 3.1.1 is released
   val VERSION310 = SparkShimVersion(3, 1, 0)
   val VERSION = SparkShimVersion(3, 1, 1)
-  val VERSIONNAMES = Seq(s"$VERSION310-SNAPSHOT", s"$VERSION", s"$VERSION-SNAPSHOT")
+  val VERSION320 = SparkShimVersion(3, 2, 0)
+  val VERSIONNAMES: Seq[String] = Seq(VERSION310, VERSION, VERSION320)
```

Comment:
I would like to see `3.2.0` added as a separate shim jar because once `3.1.1` is released (which should be soon) we will want to add `3.1.1` to the jars we include in the official output, but not `3.2.0-SNAPSHOT`